src/PlotRecipes/Potentials.jl

```diff
@@ -208,26 +212,30 @@ function get_crosssection_idx_and_value(grid::CartesianGrid3D{T}, x::Union{Real,
     cross_section, idx, value, units
 end
 
-@recipe function f(sp::ScalarPotential{T,3,Cartesian}; x = missing, y = missing, z = missing, contours_equal_potential = false) where {T <: SSDFloat}
+@recipe function f(sp::ScalarPotential{T,3,Cartesian}; x = missing, y = missing, z = missing, contours_equal_potential = false, all_point_types_bits= false) where {T <: SSDFloat}
```

Comment:
Or, if you make this a keyword argument...
```suggestion
@recipe function f(sp::ScalarPotential{T,3,Cartesian}; x = missing, y = missing, z = missing, contours_equal_potential = false; all_point_types_bits= false) where {T <: SSDFloat}
```